cartographer/mapping/internal/2d/local_trajectory_builder_2d.cc

```diff
@@ -206,6 +203,13 @@ LocalTrajectoryBuilder2D::AddRangeData(
   return nullptr;
 }
 
+sensor::PointCloud LocalTrajectoryBuilder2D::RunAdaptiveVoxelFilter(
```

Comment:
Extracting this one liner into a function does not improve readability. Inline below? If you wanted to, you could refactor the `AdaptiveVoxelFilter` which only keeps the option around and seems to be only ever used for a single call. This would be a follow up PR though: Make the function visible in the voxel filter header and remove the `AdaptiveVoxelFilter` class.